### PR TITLE
DCOS-13205: Clean up if-else statements for readability

### PR DESCRIPTION
**Pull request by jfurrow**
This PR is meant to clean up the readability of the declined offers copy logic, as requested here: https://github.com/dcos/dcos-ui/pull/1721/files#r95681321
and here: https://github.com/dcos/dcos-ui/pull/1721/files#r95681803

**Checklist**
- [ ] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

**Comment by jfurrow** [comment]
@mlunoe I hope you don't mind but I assigned you since you commented on this originally <3

**Comment by nLight** [comment]
run unit tests

File: plugins/services/src/js/containers/service-debug/ServiceDebugContainer.js [hunk 1]
```diff
@@ -37,15 +37,29 @@ class ServiceDebugContainer extends React.Component {
...
+  getDeclinedOffersTable() {
+    const {service} = this.props;
+
+    if (this.isFramework(service)) {
+      return null;
+    }
+
+    const queue = service.getQueue();
+
+    if (!DeclinedOffersUtil.shouldDisplayDeclinedOffersWarning(service)
```
Comment (jfurrow) [inline plugins/services/src/js/containers/service-debug/ServiceDebugContainer.js]: Good catch, done!

**Comment by jfurrow** [comment]
Thanks @mlunoe, I had neglected to check for the `offers` data before attempting to render the table. I implemented the proper logic and it should be fixed now!

**Comment by jfurrow** [comment]
run integration tests

**Comment by jfurrow** [comment]
Ping @mlunoe

**Comment by mlunoe** [comment]
Sorry for the wait! Looks good! :shipit: !

**Pull request merged**